tests/Unit/Evolution/Systems/CurvedScalarWave/CMakeLists.txt

```diff
@@ -15,5 +16,5 @@ add_test_library(
   ${LIBRARY}
   "Evolution/Systems/CurvedScalarWave/"
   "${LIBRARY_SOURCES}"
-  "CurvedScalarWave;CurvedScalarWaveHelpers;GeneralRelativityHelpers"
+  "DataStructures;CurvedScalarWave;CurvedScalarWaveHelpers;GeneralRelativityHelpers;Utilities"
```

Comment:
[optional] You can also change this to
```
  ""
  )

target_link_libraries(
  ${LIBRARY}
  PRIVATE
  # all the libs to link
  )
```
this is easier to read and what we're hoping to switch to in the long term :)

Reply:
changed